pkg/goDB/encoder/encoders/encoders.go

```diff
@@ -8,20 +8,24 @@ import (
 // Type denotes the type of encoder
 type Type int
 
-// Enumeration of directions to be considered
+// IMPORTANT:
+// When implementing new encoders, make sure to add the type above MaxEncoderType. Otherwise, compatibility
+// with existing databases is broken
 const (
-	EncoderTypeLZ4  Type = iota // LZ4 encoder / compressor (default, hence allocated the value 0)
-	EncoderTypeNull             // Null encoder
-	EncoderTypeZSTD             // ZSTD encoder / compressor
+	EncoderTypeLZ4Custom Type = iota // LZ4 encoder / compressor with custom checksum stripping (default, hence allocated the value 0)
```

Comment:
The beauty of renaming `EncoderTypeLZ4` to `EncoderTypeLZ4Custom` is that the `defaultEncoderType` in `gpfile` does not have to be changed. It will now point to the updated implementation of LZ4.